.github/workflows/test-deploy.yml

```diff
@@ -104,7 +104,12 @@ jobs:
         with:
           registry-url: https://registry.npmjs.org
       - run: npm install
-      - run: npm ${{ inputs.release && 'publish' || 'publish --dry-run' }}
+      - run: |
+          if [$GITHUB_RUN_ATTEMPT == 1 ]; then
```

Comment:
er... what about checking existence on the npm pkg registry instead of checking `GITHUB_RUN_ATTEMPT`?

Scenario: an earlier step in this workflow is flaky & fails, you rerun the workflow, then everything succeeds but the npm pkg is silently not published.

Reply:
too slow, already been changed

Reply:
https://github.com/iterative/cml/pull/1339